main.go

```diff
@@ -233,6 +235,17 @@ func parseCategory(filePath string, categorName string) SharedCategory {
 		toml.Unmarshal(byteValue, &sharedCategory)
 		return sharedCategory
 	}
+	if fileExtension == ".yaml"{
+		yamlFile, err := ioutil.ReadFile(filePath)
+		if err != nil {
+			fmt.Println(err)
+			return SharedCategory{}
+		}
+		byteValue, _ := yaml.Marshal(yamlFile)
```

Comment:
I tried using a yaml file and the struct was empty.
I think it has to do with this.
Don't need to Marshal the file to bytes to Unmarshal yaml into the struct.
You can just Unmarshal the `yamlFile`.

Reply:
resolved